### ISSUE TITLE
Not working on Arch + Gnome 3.24.2

### ISSUE THREAD

**Issue by dvorapa** [numixproject/numix-folders#175]
GUI:
```
> sudo numix-folders
No protocol specified
Unable to init server: Could not connect: Connection refused
No protocol specified
Unable to init server: Nelze se připojit: Connection refused
No protocol specified
Unable to init server: Nelze se připojit: Connection refused

(numix-folders:2180): Gtk-WARNING **: cannot open display: :0
```
Terminal:
Reported in #160

**Comment by ludenticus** [comment]
This isn't a bug. It is the intended behavior of Wayland. Run `sudo numix-folders --cli` instead. Or login with `xorg`.

**Comment by palob** [comment]
Can be closed in favour of #162.
